data/json/mutations/mutations.json

```
@@ -1033,9 +1033,10 @@
     "type": "mutation",
     "id": "GOODMEMORY",
     "name": { "str": "Good Memory" },
-    "points": 3,
+    "points": 0,
     "description": "You have an exceptional memory, and find it easy to remember things.  Your skills will erode slightly slower than usual.",
```

```suggestion
    "description": "You have an exceptional memory, and find it easy to remember things. However, you don't benefit as much as other people from droping a subject for a while and will thus end up learning slower.",
```

We need to also explain why this is a mixed bag

---

Edited your message to fix a typo in your suggestion